Gruntfile.js

```diff
@@ -9,6 +9,9 @@ module.exports = function (grunt) {
 			options: {
 				ignoreCompilerErrors: true, // Remove this once compile errors are resolved
 			}
+		},
+		intern: {
+			version: 4
```

Comment:
We don't need this anymore with your new grunt-dojo2 changes, right?

Reply:
`grunt-dojo2` still defaults to Intern 3 for backwards compatibility, so we need this